force-app/main/educationCloudSettings/lwc/edcReleaseManagementCard/edcReleaseManagementCard.css

```diff
@@ -0,0 +1,30 @@
+.progress-container {
+    height: 30px;
+}
+.progressring-container {
+    max-width: 30px;
+}
+.progressring-content {
+    vertical-align: middle;
+}
```

Comment:
These classes are additional non lwc-token classes that we are adding into the design. I'm calling them out as typically we do want to avoid them. In particular, the pixel-specific widths are things I would prefer to avoid from a technical perspective. The relative vertical alignment may have a design token we can lean back on.

Reply:
Changed to relative units (em). Explicit height is needed here to avoid UI jumping when all release gates are finished loading